index.js

```diff
@@ -0,0 +1,6 @@
+/* jshint node: true */
+'use strict';
+
+module.exports = {
+  name: 'model-fragments'
```

Comment:
Crap, does this have to be `model-fragments` to match the module name? I regret not using the full `ember-data-model-fragments` already :grimacing: